### DISCUSSION TITLE
with self signed using mkcert  permanently delete document : unable to verify the first certificate

### DISCUSSION THREAD

**Discussion by isafe** [outline/outline#4304] [Self Hosting] [answered]
1. permanently delete document, error 500
2. docker logs outline 

`Error: unable to verify the first certificate`

**Expected behavior**

**Screenshots**
If applicable, add screenshots or videos to help explain your problem.

**Outline (please complete the following information):**
- Install: [self hosted, minio]
- Version: [0.66.1]

**Desktop (please complete the following information):**
 - Browser [chrome]

<img width="609" alt="image" src="https://user-images.githubusercontent.com/5019002/196041914-078c4a25-bd6e-47f8-a099-2b1dcde8292b.png">

**Comment by isafe** [comment, marked as answer]
solution:
set environment  `NODE_TLS_REJECT_UNAUTHORIZED=0`

**Comment by goodnull** [comment]
Can it be added to the .env.sample file?

**Reply by tommoor** [reply]
Probably not, because it's bad practice to set this – you're literally telling Node not to verify SSL certificates.
